### ISSUE TITLE
learning rate tuning

### ISSUE THREAD

**Issue by pengzhangzhi** [HKUNLP/Dream#12]
Hi authors,

You mentioned in the blog that fine-tuning from an ARM requires careful tuning of the LR. I would like to learn a bit of details about this. Would you give a summary of how is the LR tuning conducted?

**Comment by jiacheng-ye** [comment]
Hi there, we experimented with various learning rates, ranging from 1e-6 to 1e-3, and compared the resulting learning curves, e.g., training loss.

**Comment by pengzhangzhi** [comment]
thanks!! what lr did u use at last?

**Comment by jiacheng-ye** [comment]
We have recently released our fine-tuning script. Check it out!
